nginx/nginx.conf

```diff
@@ -34,28 +34,19 @@ http {
   client_header_timeout 60s;
   send_timeout 60s;
 
-  include  mime.types;
+  include mime.types;
   include /etc/nginx/useragent.rules;
 
-  map $sent_http_content_type $expires {
-    default                    off;
-    text/html                  epoch;
-    text/css                   max;
-    application/javascript     max;
-    ~image/                    max;
-  }
-
   server {
-    listen 8080;
-    root /usr/src/app/dist;
+    listen 80;
+    root /usr/share/nginx/html/;
     autoindex off;
     access_log /dev/stdout;
     gzip on;
-    gzip_types application/javascript text/html text/css application/json;
+    gzip_types application/javascript text/css application/json;
```

Comment:
Was there a problem with maintaining compression on html?

Reply:
It created a warning when starting the server, `duplicate MIME type 'text/html' in /etc/nginx/nginx.conf:46; text/html is included by default`. It was because `text/html` is included by default, so removing it is ok.